src/listen/websocket.rs

```diff
@@ -333,26 +409,35 @@ where
                         let mut write = write_clone.lock().await;
                         if let Err(e) = write.send(keep_alive_message).await {
                             eprintln!("Error Sending Keep Alive: {:?}", e);
+                            let _ = event_tx_keep_alive
+                                .send(Event::Error(DeepgramError::from(e)))
+                                .await;
                             break;
```

Comment:
TBD (in a later PR): we should only send keepalives when we haven't sent any other messages in a given time frame.

Reply:
The server will close the connection if it never receives any audio even if you send keep alives